### ISSUE TITLE
Comment children are not nested

### ISSUE THREAD

**Issue by BertmV** [timber/timber#954]
### Expected behavior

When I iterate through post.comment.children I expect those to be properly nested (so a reply to a reply would be placed in post.comment.children.children)
### Actual behavior

All replies, including those in reply to a reply, are at the top level (post.comment.children)
### Steps to reproduce behavior

```
{# comment-thread.twig #}
<div class='module-container comments'>
  <section class='module comments' aria-label=''>
    <header>
      <h1>Comments</h1>
    </header>
    {% if post.comment_count > 0 %}
      {% for comment in post.comments %}
        {% set comment_class = "start-topic" %}
        {% include "elements/modules/comment.twig" %}
      {% endfor %}
    {% endif %}
    <footer>
      [New Comment]
    </footer>
  </section>
</div>
```

```
{# comment.twig #}
  <article class='module comment {{ comment_class }}' aria-label=''>
    <header>
      {{ comment.author.name }}
    </header>
    <div>
      {{ comment.content }}
    </div>
    <footer></footer>
    {% for comment in comment.children %}
      {% set comment_class = "reply" %}
      {% include "elements/modules/comment.twig" %}
    {% endfor %}
  </article>
```

This renders all replies (children) as siblings instead of descendants.
### What version of WordPress, PHP and Timber are you using?

WordPress 4.5, PHP 7, Timber 0.22.5
### How did you install Timber?

Upgraded to newest version via plugin updater in WordPress dashboard

**Comment by shackep** [comment]
I am also having this issue. Initially I was only getting parents. I created a macro that referred to itself if the current comment had children to create depth. I can only get comments two levels deep.

**Comment by shnam7** [comment]
Yes, I'm also experiencing the same issue!

**Comment by Eworm** [comment]
Same thing happening to me!

**Comment by jarednova** [comment]
I believe this is closed by #1205 — the current version now has nested comments
